page.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html>
+<head> 
+  <link rel="stylesheet" type="text/css" href="CSS/null.css"> 
+  <link rel="stylesheet" type="text/css" href="CSS/footer.css">
+  <link rel="stylesheet" type="text/css" href="CSS/header.css">
+  <link rel="stylesheet" type="text/css" href="CSS/html-body.css">
+  <link rel="stylesheet" type="text/css" href="CSS/icons-style.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Open+Sans&family=Roboto&display=swap" rel="stylesheet">
+  <meta charset="utf-8">  
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+</head>
+<body>
+  <footer class="footer">
+    <div class="footer__body">
+    <div class="footer__logo-container">
+      <h2 class="logo">Cakes</h2>
+    </div>
+    <nav class="footer__navigation-list">
+      <ul class="navigation-list">
+        <li class="list__item">
+          <a class="list__link" href="#our-cakes">Наши торты</a>
+        </li>
+        <li class="list__item">
+          <a class="list__link" href="#personal-order">Индивидуальный заказ</a>
+        </li>
+        <li class="list__item">
+          <a class="list__link" href="#reviews">Отзывы</a>
+        </li>
+        <li class="list__item">
+          <a class="list__link" href="#about-us">О нас</a>
+        </li>
+        <li class="list__item">
+          <a class="list__link" href="#our-advantages">Как заказать</a>
+        </li>
+        <li class="list__item">
+          <a class="list__link" href="#contacts">Контакты</a>
+        </li>
+      </ul>
+    </nav>  
+    <div class="footer__phone-number">
+      <a class="list__link" href=tel:+79068999296>+7 906 899 92 96</a>
```

Comment:
It is not a `list_link`. Phone number could be a separate bem class. As far as I remember it's also reused somewhere on design mock up.